src/core/util/env.js

```diff
@@ -8,12 +8,12 @@ export const inBrowser = typeof window !== 'undefined'
 export const inWeex = typeof WXEnvironment !== 'undefined' && !!WXEnvironment.platform
 export const weexPlatform = inWeex && WXEnvironment.platform.toLowerCase()
 export const UA = inBrowser && window.navigator.userAgent.toLowerCase()
-export const isIE = UA && /msie|trident/.test(UA)
-export const isIE9 = UA && UA.indexOf('msie 9.0') > 0
+export const isIE = inBrowser && document.documentMode
+export const isIE9 = inBrowser && document.documentMode === 9
 export const isEdge = UA && UA.indexOf('edge/') > 0
 export const isAndroid = (UA && UA.indexOf('android') > 0) || (weexPlatform === 'android')
 export const isIOS = (UA && /iphone|ipad|ipod|ios/.test(UA)) || (weexPlatform === 'ios')
-export const isChrome = UA && /chrome\/\d+/.test(UA) && !isEdge
+export const isChrome = inBrowser && window.chrome && !window.opr
```

Comment:
Have you checked [this StackOverflow's answer](https://stackoverflow.com/a/50942571/794641) (disclaimer: I haven't)?

Reply:
I haven't